guides/v2.3/magento-functional-testing-framework/2.3/getting-started.md

```diff
@@ -137,101 +140,97 @@ Example: `MAGENTO_ADMIN_USERNAME=admin`
 
-If you are installing the MFTF from inside your Magento installation, this is automatically done when you [build the project].
+If you are installing the MFTF from inside your Magento installation, this is automatically done when you [build the project][].
 
 {:.bs-callout .bs-callout-warning}
 If you do not have access to your Magento installation and cannot complete the step you will not be able to use the Magento CLI commands.
 
-### Step 6. Enable the Magento CLI commands
+### Step 7. Generate and run tests   {#run-tests}
 
-In the `magento2/dev/tests/acceptance` directory, run the following command to enable the MFTF to send Magento CLI commands to your Magento instance.
+To run tests, you need a running Selenium server and [`mftf`][] with required parameters.
 
- ```bash
-cp .htaccess.sample .htaccess
-```
-
-### Step 7. Generate existing tests     {#generate-tests}
+#### Run the Selenium server    {#selenium-server}
 
-In the `magento2/` root directory, run the following command to generate tests as PHP classes from XML files:
+Run the Selenium server in terminal.
+Example of the run for Chrome:
 
 ```bash
-vendor/bin/mftf generate:tests
+cd <path_to_directory_with_selenium_server_and_webdiver>/
+```
+```bash
+java -Dwebdriver.chrome.driver=chromedriver -jar selenium-server-standalone-3.14.0.jar
 ```
 
-### Step 8. Run tests   {#run-tests}
+If you do not have the Selenium server yet:
 
-To run tests, you need a running Selenium server and a [`codecept`] or [`mftf`] with required parameters.
+1. [Download the latest Selenium Server][].
 
-#### Run the Selenium server    {#selenium-server}
+2. [Download a Selenium web driver for your web browser][] into the same directory that contains the Selenium server.
 
-Run the Selenium server in terminal:
+#### Generate and run all tests {#run-all-tests}
 
 ```bash
-java -jar <path_to_selenium_directory>/selenium-server-standalone-<version>.jar
+vendor/bin/mftf generate:tests
+```
+```bash
+cd dev/tests/acceptance
+```
+```bash
+vendor/bin/codecept run functional
 ```
 
-If you do not have the Selenium server yet:
-
-1. [Download the latest Selenium Server].
-
-2. [Download a Selenium web driver for your web browser] into the same directory that contains the Selenium server.
+See more commands in [`codecept`][].
 
-3. Add the directory with the web driver to `PATH`.
+#### Run a simple test {#run-test}
 
-#### Run all tests {#run-all-tests}
+To clean up the previously generated tests, and then generate and run a single test `AdminLoginTest`, run:
 
 ```bash
-vendor/bin/codecept run
+vendor/bin/mftf run:test AdminLoginTest --remove
 ```
 
-See more commands in [`mftf`] and [`codecept`].
+See more commands in [`mftf`][].
 
 ### Step 9. Generate reports    {#reports}
 
-The testing reports are generated in a CLI during testing.
+The testing reports are generated in CLI during the testing.
+
+The MFTF also provides visual reports using [Allure][].
+To view the reports in GUI:
```

Comment:
Suggest clarifying the overview of the reports.  Consider:


During testing, the MFTF reporting tool generates test reports.   (I don't think this is quite right either. Maybe a link to the MTFT Reporting section)
You can generate visual representations of the report data using Allure.